iphone/Classes/TiUITableView.m

```diff
@@ -2651,6 +2699,12 @@ - (void)scrollViewDidEndDecelerating:(UIScrollView *)scrollView
 
 - (void)didDismissSearchController:(UISearchController *)searchController
 {
+#if IS_XCODE_9
+  if ([TiUtils isIOS11OrGreater]) {
+    tableview.frame = CGRectMake(0, 0, self.frame.size.width, self.frame.size.height);
+  }
```

Comment:
I don't feel save with this. What if the table is not positioned full screen? And what about orientation changes? Please add tests for those scenarios.

Reply:
Table's contentOffset should be managed rather table view frame. Otherwise view below to tableview will get visible. So managed contentOffset rather frame.